src/components/DonateButton.jsx

```diff
@@ -1080,23 +1080,44 @@ export default class Root extends React.PureComponent {
 
   afterSuccessfulDonate() {
     const { donateToDefaultDac } = this.state;
+    const { customThanksMessage } = this.props.model;
```

Comment:
`customThanksMessage` would contain the message. In this case "Thank you for your donation to the Commons Stack! If you...."

Reply:
What is the value of customThanksMessage in the case of normal ThanksMessage?
null or No??

I want to write a condition on it?

Reply:
I expect to fetch the following content from feathers. Please put it in MongoDB.


"Thank you for your donation to the Commons Stack! If you would like to receive <a href='https://medium.com/commonsstack/cstk-the-token-of-the-commons-stack-trusted-seed-931978625c61'>CSTK</a> tokens, please <a href= 'mailto:info@commonsstack.org?subject=I just donated to Commons Stack!' > contact us</a> and we'll send you all the details!"

Reply:
@aminlatifi  check my new commit